apps/android/app/src/main/res/values-de/strings.xml

```diff
@@ -4,73 +4,73 @@
-  <string name="profile_will_be_sent_to_contact_sending_link">Your profile will be sent to the contact that you received this link from.</string>
-  <string name="connect_via_link_verb">Connect</string>
+  <string name="connect_via_contact_link">Über den Kontakt-Link verbinden?</string>
+  <string name="connect_via_invitation_link">Über den Einladungs-Link verbinden?</string>
+  <string name="profile_will_be_sent_to_contact_sending_link">Dein Profil wird an den Kontakt gesendet von dem Du diesen Link erhalten hast.</string>
+  <string name="connect_via_link_verb">Verbinden</string>
 
   <!-- Server info - ChatModel.kt -->
-  <string name="server_connected">connected</string>
-  <string name="server_error">error</string>
-  <string name="server_connecting">connecting</string>
-  <string name="connected_to_server_to_receive_messages_from_contact">You are connected to the server used to receive messages from this contact.</string>
-  <string name="trying_to_connect_to_server_to_receive_messages_with_error">Trying to connect to the server used to receive messages from this contact (error: <xliff:g id="errorMsg">%1$s</xliff:g>).</string>
-  <string name="trying_to_connect_to_server_to_receive_messages">Trying to connect to the server used to receive messages from this contact.</string>
+  <string name="server_connected">Verbunden</string>
+  <string name="server_error">Fehler</string>
+  <string name="server_connecting">verbinde</string>
+  <string name="connected_to_server_to_receive_messages_from_contact">Du bist mit dem Server verbunden, der für den Empfang von Nachrichten mit diesem Kontakt genutzt wird.</string>
+  <string name="trying_to_connect_to_server_to_receive_messages_with_error">Beim Versuch die Verbindung mit dem Server aufzunehmen, der für den Empfang von Nachrichten mit diesem Kontakt genutzt wird, ist ein Fehler aufgetreten (Fehler: <xliff:g id="errorMsg">%1$s</xliff:g>).</string>
+  <string name="trying_to_connect_to_server_to_receive_messages">Versuche die Verbindung mit dem Server aufzunehmen, der für den Empfang von Nachrichten mit diesem Kontakt genutzt wird.</string>
 
   <!-- Item Content - ChatModel.kt -->
-  <string name="deleted_description">deleted</string>
-  <string name="sending_files_not_yet_supported">sending files is not supported yet</string>
-  <string name="receiving_files_not_yet_supported">receiving files is not supported yet</string>
-  <string name="sender_you_pronoun">you</string>
-  <string name="unknown_message_format">unknown message format</string>
-  <string name="invalid_message_format">invalid message format</string>
+  <string name="deleted_description">Gelöscht</string>
+  <string name="sending_files_not_yet_supported">Das Senden von Dateien wird aktuell nicht unterstützt</string>
+  <string name="receiving_files_not_yet_supported">Der Empfang von Dateien wird aktuell nicht unterstützt</string>
+  <string name="sender_you_pronoun">Du</string>
```

Comment:
Maybe, "Sie"?

Or is it common to use "Du" in the apps? In Russian translations we use the equivalent of "Sie"